public/api/conf/3.0/application.raml

```diff
@@ -389,3 +389,47 @@ documentation:
                       <code>INTERNAL_SERVER_ERROR</code>
                       <message>Internal server error</message>
                   </errorResponse>
+
+  /status-request:
+    /mrn:
+      /{mrn}:
+        uriParameters:
+          mrn:
+            description: Movement Reference Number
+            type: string
+            example: "ABC123"
+        get:
+          displayName: Request the status of Declaration
+          description: Allows requesting the status of a Declaration.
+          headers:
+            Accept:
+              description: Specifies the version of the API that you want to call. See [versioning](/api-documentation/docs/reference-guide#versioning).
+              type: string
+              required: true
+              example: application/vnd.hmrc.3.0+xml
+            Content-Type:
+              description: Specifies the format of the request body, which must be XML. UTF-8 is the only valid character set.
+               Download [schemas and example messages](/api-documentation/docs/api/download/customs-declarations/2.0/wco-declaration-schemas.zip).
+              type: string
+              required: true
+              example: application/xml; charset=UTF-8
+            X-Badge-Identifier:
+                description: Represents the end-user who is interacting with the subscribed application. Required for CSP users. Must be numeric or upper case letters with a minimum length of 6 and a maximum length of 12.
+                type: string
+                required: true
+                example: ABC123
+          (annotations.scope): "write:customs-declaration"
+          securedBy: [ sec.oauth_2_0: { scopes: [ "write:customs-declaration" ] } ]
+          responses:
+            202:
```

Comment:
response is 200

Reply:
a reference to an example body should be included -
api/conf/3.0/schemas/wco/status/StatusQueryResponseExample.xml